Assets/Tests/InputSystem/APIVerification.cs

```diff
@@ -0,0 +1,46 @@
+#if UNITY_EDITOR
+using System;
+using System.Linq;
+using NUnit.Framework;
+using Mono.Cecil;
+using Mono.Collections.Generic;
+
+partial class APIVerification
```

Comment:
Preferably tag "Tests" as a postfix to the class (and file name). IMO nice to immediately spot in file and type requesters what's test stuff and what's "real" stuff.